examples/gltf/app.js

```diff
@@ -203,22 +215,32 @@ export class DemoApp {
   }
 
   onRender({gl, time, width, height, aspect}) {
-    gl.viewport(0, 0, width, height);
     clear(gl, {color: [0.2, 0.2, 0.2, 1.0], depth: true});
 
+    this.vrDisplay.getFrameData(this.frameData);
+
+    gl.viewport(0, 0, width * 0.5, height);
+    this.onRenderEye({gl, time}, 'left');
+
+    gl.viewport(width * 0.5, 0, width * 0.5, height);
+    this.onRenderEye({gl, time}, 'right');
+  }
+
+  onRenderEye({gl, time}, eye) {
     const [pitch, roll] = this.rotation;
     const cameraPos = [
       -this.translate * Math.sin(roll) * Math.cos(-pitch),
       -this.translate * Math.sin(-pitch),
       this.translate * Math.cos(roll) * Math.cos(-pitch)
     ];
 
-    const uView = new Matrix4()
-      .translate([0, 0, -this.translate])
-      .rotateX(pitch)
-      .rotateY(roll);
+    const uView = new Matrix4(Array.from(this.frameData[`${eye}ViewMatrix`]));
+      // .translate([0, 0, -this.translate])
+      // .rotateX(pitch)
+      // .rotateY(roll);
 
-    const uProjection = new Matrix4().perspective({fov: radians(40), aspect, near: 0.1, far: 9000});
+    // const uProjection = new Matrix4().perspective({fov: radians(40), aspect, near: 0.1, far: 9000});
+    const uProjection = new Matrix4(Array.from(this.frameData[`${eye}ProjectionMatrix`]));
```

Comment:
Same as above. Don't need to make the copies.

Reply:
Same as above.